### DISCUSSION TITLE
PROBLEMA SCRUTINIO TRIMESTRE

### DISCUSSION THREAD

**Discussion by gianluca3090** [iisgiua/giuaschool#610] [Richieste di aiuto]
Buongiorno ,ieri nella procedura di scrutinio di una classe del TIMESTRE dopo aver fatto l'aggiornamento , mi usciva error 500.
Mi potreste aiutare?
Grazie

**Comment by trinko** [comment]
Devi fare una segnalazione d'errore completa.

[Come fare una segnalazione d'errore](https://github.com/iisgiua/giuaschool/discussions/576)
